fixieai/agents/api.py

```diff
@@ -20,6 +22,28 @@ class Embed:
     # upload an Embed object to Fixie and generate a URL.
     uri: str
```

Comment:
How would the SDK user upload their bytes to a uri?

Reply:
oops, that was an old comment. Rewritten.

Reply:
I didn't mean the comment, but functionality-wise. As an SDK user, I have an image or a large amount of text that I want to pass to Fixie. Say I have a Func that parses a PDF or gets a text from a custom endpoint.

Reply:
just set `.content` and it will flow back to our system as a data URI, at which point we will upload it to GCS

Reply:
oh that's interesting. Then we don't need the sha hashing as well because the uri will remain the same for the same embeds. I think one fallback is that data will be accessible via a public url but not something to worry about right now.